### ISSUE TITLE
Backbone not freezed during training of classifier 

### ISSUE THREAD

**Issue by wuchlei** [BestJuly/IIC#5]
I see you didn't freeze the parameters of backbone during the classifier training, which is different from image SSL papers. I wonder is there any reason for this?

**Comment by BestJuly** [comment]
Hi, @wuchlei . This is a good question and actually I have the same question as you do.

In my experiments, I follow previous works about the fine-tuning part, where the backbone is not frozen.

In my point of view, I think the reason may be 
1) If freezing backbone parameters, the fine-tuning performance is not good, compared to fine-tuning all parameters or train from scratch (I have tried this). And for now, the improvements can be brought by using video SSL methods and use the SSL model as an initialization method.
2) Few works in video-based SSL methods use this setting. Obtaining these results for one proposal is simple but it might be hard to compare with a lot of previous works.

Therefore, I think retrieval is also an important evaluation metric to see the performance of a SSL method. However, there are still a lot of work only reporting final results from video recognition, where all parameters are fine-tuned. Actually, there are two training steps, SSL training and fine-tuning. Therefore, it is hard to say which part brought the most improvements for these methods. 

For image SSL papers, retrieval will also be conducted using different conv layer features, as well as fine-tuning only the last linear layer, which I think seems more reasonable.

**Comment by wuchlei** [comment]
Thanks for explanation!
